pkgs/development/libraries/libfido2/default.nix

```diff
@@ -7,30 +8,45 @@
 , libcbor
 , openssl
 , udev
+, zlib
 }:
 
 stdenv.mkDerivation rec {
   pname = "libfido2";
-  version = "1.5.0";
+  version = "1.7.0";
   src = fetchurl {
     url = "https://developers.yubico.com/${pname}/Releases/${pname}-${version}.tar.gz";
```

Comment:
Any reason why we are not fetching from https://github.com/Yubico/libfido2/releases/tag/1.7.0 ?

Reply:
Releases on https://developers.yubico.com/libfido2/Releases/ are signed

Reply:
Are we checking that?

Reply:
not automatically, but I do that manually